backend/src/__tests__/userRoute.test.js

```diff
@@ -67,3 +68,30 @@ describe('User authentication flow', () => {
     });
   });
 });
+
+describe('User authentication flow', () => {
+  it('should create a new user', async () => {
+    const res = await request(app)
+      .post('/users/create')
+      .send({
+        email: 'demo1@gmail.com',
+        password: 'harish',
+      });
+    expect(res.statusCode).toEqual(200);
+    expect(res.body).toHaveProperty('user');
+    const { token } = res.body;
+
+    const resAddAuthToken = await request(app)
+      .post('/user/add-token')
+      .set('Authorization', `Bearer ${token}`)
+      .send({
+        service: 'github',
+        token: GITHUB_TEST_AUTH_TOKEN,
+      });
+    expect(resAddAuthToken.statusCode).toEqual(200);
+
+    const resDelete = await request(app)
```

Comment:
a better place to clean all data would be `afterall` method

Reply:
Yes, I started with a simple test which worked fine with this technique. But as the tests are growing, it's not scaling well, I need to start using the hook methods.